### ISSUE TITLE
[Bug]: Several bugs in JASP on Linux Ubuntu 20.4 LTS

### ISSUE THREAD

**Issue by KrisKawski** [jasp-stats/jasp-issues#1751]
### JASP Version

0.16.3

### JASP Module

jaspJags

### What analysis are you seeing the problem on?

JAGS, Export Results (of CFA), blurred font

### What OS are you seeing the problem on?

Linux Ubuntu 20.04 LTS “Focal Fossa”

### Bug Description

Issues:

1) JAGS - The JAGS module is not working for me - when I open any database and run the JAGS module, the JASP program always closes itself (JASP 0.16.3 on Linux Ubuntu 20.04),

2) Export Results - several times after clicking "Export Results" (after completing the data analysis) and selecting "Desktop" as the destination, JASP crashed and I had to restart the computer,

3)  Blurred font in the "Results" section (on the right side) when I start analysis (e.g. SEM and others) - is slightly blurred
![Blurred font (on the right)](https://user-images.githubusercontent.com/107765783/174456350-b8730bb1-6615-4402-9494-937a73486ff6.png)

### Expected Behaviour

Generally speaking - when I was using a previous version of JASP (0.16.0, 0.16.1 or 0.16.2 - I don't remember which one was available for Linux - probably version 0.16.1) - everything worked fine.

Do any of you have an older version (0.16.1) available for Linux Ubuntu? Is it even possible to go back to an older version?

Or maybe with newer versions of JASP (i.e. 0.16.3+) I won't have these problems?

### Steps to Reproduce

1. Unclear font on right side of screen:  ("Results" section) all the time

2. Problems with JAGS module: Show main menu → Open → Recent files → I select a database → Show modules menu → JAGS → click on the JAGS module icon → effect: immediate and automatic exit from JASP

3. Problems with export results: after the CFA analysis is performed and the results are displayed → Results → Export Results → selecting the "Desktop" option → effect: JASP software hangs (freezes)
 
...

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if any.
- [ ] I have included a JASP file that causes the crash/bug, if any.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by juliuspf** [comment]
@KrisKawski, thanks for the report. Any ideas @vandenman?

**Comment by vandenman** [comment]
> Problems with JAGS module: Show main menu → Open → Recent files → I select a database → Show modules menu → JAGS → click on the JAGS module icon → effect: immediate and automatic exit from JASP

@KrisKawski could recreate the crash and then post the log files after the crash happened? You may need to enable logging first like so:
![image](https://user-images.githubusercontent.com/21319932/174599190-b7a3fbc7-8e29-43ee-b9d5-7adccd6025e8.png)

If `Show logs` doesn't work for you (it doesn't for me), you can also find them in `~/.var/app/org.jaspstats.JASP/data/JASP/JASP/Logs`. There should be two log files of interest, `'JASP <date> <time> Engine 1.log'` and `'JASP <date> <time> Desktop.log'`.

> Unclear font on right side of screen: ("Results" section) all the time

@KrisKawski Do you use the same font for the `Interface` and `Results & Help`? (It looks like you do, but I'd like to double-check). Also, does the issue persist when you enable `Safe graphics mode`? You can enable this here:
![image](https://user-images.githubusercontent.com/21319932/174600121-2c2ea192-6e8a-4594-8dcb-5f84017e22e8.png)



@JorisGoosen do you have any other ideas for issues 1 and 3? For 1 it smells like the WebEngine isn't working great, since the fonts for the QML input are rendered fine. This could be a regression in Qt6.

**Comment by JorisGoosen** [comment]
@KrisKawski 
> Do any of you have an older version (0.16.1) available for Linux Ubuntu? Is it even possible to go back to an older version?

Yes you can go back indeed, it is not exactly fun but should help you work with jags until we figure out the problem.
`flatpak remote-info --log flathub org.jaspstats.JASP`
that should get you something that looks like this: https://github.com/flatpak/flatpak/issues/3097#issuecomment-530583469
Then you get the right "commit" for the 0.16.1 version (I dont have linux here right now so I cant check it for you)
and fill that in here:
`flatpak update --commit=<commit-number-here> org.jaspstats.JASP`

As far as 3 goes, this looks similar to some other font bugs. I guess changing the font would be a good idea as well as making sure the interface-zoom is at 100% (by pressing `ctrl+0`)

Why `jags` won't run... Maybe some library is linked wrong?
@vandenman could try starting `flatpak run --devel --command=sh org.jaspstats.JASP` and see if `jags` runs manually.
If you don't have it installed yet I can recommend `flatpak install flathub org.jaspstats.JASP.Debug` 
Otherwise we can try running `jasp` in debugmode in that development prompt and see where it crashes.

**Comment by KrisKawski** [comment]
Dears Julius, Don and Joris, 

thank you very much for all your advice.

It looks like everything is already working in my JASP. 

How? I've selected the following settings: Show main menu → Preferences → Interface → User Interface Options → Fonts: FreeSans [GNU] (same font everywhere: in Interface, in R, JAGS, or lavaan code, and in Result & help).
And then: Miscellaneous options → I've reduced the Zoom (to 140) and the Scroll Speed (to 800) → and I've selected Safe graphics mode.
The font is now clear and the JAGS mode - it works!

Again, thank you so much for your help and I wish you all the best. 
Best regards, Kris
